Athena's Wrath/Athena's Wrath/main.swift

```diff
@@ -0,0 +1,355 @@
+    switch riddleChoice {
+    case 76:
+        print("You were wrong. Try again or Medusa will have your head!")
+        riddleAnswer = readLine()
+        
+        riddleChoice = Int(riddleAnswer!) ?? 76
+        riddleChoice = Int(riddleAnswer!) ?? 67
+        riddleChoice = Int(riddleAnswer!) ?? 69
+    case 67:
+        print("Congratulations! You chose correctly! You were spared from Medusa's wrath.")
+        print("""
+
+MEDUSA: You have bested me. I leave you with a parting gift. I give you my blessing, Athena.
+
+""")
+        print("You have gained: \(medusasBlessing)")
+        rTrue = false
+    case 69:
+        print("You chose incorrectly, please make another choice.")
+        riddleAnswer = readLine()
+        
+        riddleChoice = Int(riddleAnswer!) ?? 76
+        riddleChoice = Int(riddleAnswer!) ?? 67
+        riddleChoice = Int(riddleAnswer!) ?? 69
+        rTrue = false
+    default:
+        print("Please make a choice.")
+        riddleAnswer = readLine()
+        
+        riddleChoice = Int(riddleAnswer!) ?? 76
+        riddleChoice = Int(riddleAnswer!) ?? 67
+        riddleChoice = Int(riddleAnswer!) ?? 69
+        rTrue = false
+        }
+} while rTrue == true
+    
+sleep(1)
+
+print("""
+
+MEDUSA bows to you. She has given up and has chosen peace among you.
+
+You are the Goddess of WAR and WISDOM. It is your chance to act. MEDUSA is vulernable.
+
+Will you choose WAR or WISDOM?
+
+""")
+
+var finalAns = readLine()?.lowercased()
+
+var final = (wisdom: "wisdom", war: "war")
+
+if finalAns == final.wisdom {
+    print("""
+
+You have chosen wisdom. You mended your broken relationship with MEDUSA. MEDUSA switched sides and started fighting off the other monsters along with you. Mount Olympus was rid of the danger, with help from your new found alliance. The balance was restored.
+
+You have won, Athena; in more ways than one.
+
+
+                                            CONGRATS! YOU HAVE BEATEN THE GAME!
+
+""")
+} else if finalAns == final.war {
+    print("""
+        
+        You have chosen war. You pulled out your weapon and cut MEDUSA's head clean off. The venomous blood from MEDUSA's body splattered onto your skin... and seeped in.
+        
+        The toxins slowly entered your blood stream and drove you more and more insane by each passing minute. You realized you liked it. The killing. The dark magic transformed your trophies: \(Minotaur), \(Cerebus), \(medusasBlessing) into a dark weapon.
+        
+        Please, choose a number: 1 or 2
+        """)
+    let eChoice = Int(readLine()!)
+    
+    if eChoice == 1 {
+        print("""
+You have recieved a Dual Wielding Sword.
+
+""")
+    } else if eChoice == 2 {
+        print("""
+You have recieved a grotesque Speargun.
+
+""")
+    }
+    sleep(1)
+    
+    print("""
+
+The insanity spreads through your body and mind, like a vicious plague. You grip your weapon tightly and...
+
+""")
+    sleep(2)
+    
+        for damage in 1...7{
```

Comment:
This line of code accounts for both your range and for loop requirements but it doesn't really add to your game, so although you used it, I would like to see you put it to better use next time ;)